New settings screen

sirlucasm opened the pull request:
- Added the sweetlert2 cdn
- New screen of settings menu
- missing time change in settings menu

Commit:
added setting area

- new setting button
- added the setting function, its only add the configTimer screen

Signed-off-by: Lucas Matheus <mac@MacBook-Pro.local>

Commit:
added setting area

- new setting button
- added the setting function, its only add the configTimer screen

Signed-off-by: Lucas Matheus <mac@MacBook-Pro.local>

Commit:
added the settings screen

Commit:
Merge branch 'master' of github.com:fire4dev/roga-timer into develop

* 'master' of github.com:fire4dev/roga-timer:

Commit:
Date and Hour added

Signed-off-by: Lucas Matheus <mac@MacBook-Pro.local>

Commit:
New Features Added

- The sound effects on countdown has been added
- Date and time on top configurated

Signed-off-by: Lucas Matheus <mac@MacBook-Pro.local>

sirlucasm commented:
boraaa sider comente meu PR

siderlan commented:
Hi @fire4dev,

Your code has some lint errors. See the deploy log and fix the errors, please.

```
6:38:41 PM: Executing user command: npm run build
6:38:41 PM: > roga-timer@0.1.0 build /opt/build/repo
6:38:41 PM: > vue-cli-service build
6:38:42 PM: -  Building for production...
6:38:52 PM:  ERROR  Failed to compile with 1 errors9:38:52 PM
6:38:52 PM:  error  in ./src/views/Timer.vue
6:38:52 PM: Module Error (from ./node_modules/eslint-loader/index.js):
6:38:52 PM: error: Unexpected console statement (no-console) at src/views/Timer.vue:175:11:
6:38:52 PM:   173 |           newShortBreak = document.getElementById('swal-input2').value
6:38:52 PM:   174 |           newLongBreak = document.getElementById('swal-input3').value
6:38:52 PM: > 175 |           console.log(newFocusTime)
6:38:52 PM:       |           ^
6:38:52 PM:   176 |           console.log(newShortBreak)
6:38:52 PM:   177 |           console.log(newLongBreak)
6:38:52 PM:   178 |           if (formValues) {
6:38:52 PM: error: Unexpected console statement (no-console) at src/views/Timer.vue:176:11:
6:38:52 PM:   174 |           newLongBreak = document.getElementById('swal-input3').value
6:38:52 PM:   175 |           console.log(newFocusTime)
6:38:52 PM: > 176 |           console.log(newShortBreak)
6:38:52 PM:       |           ^
6:38:52 PM:   177 |           console.log(newLongBreak)
6:38:53 PM: failed during stage 'building site': Build script returned non-zero exit code: 1
6:38:52 PM:   178 |           if (formValues) {
6:38:52 PM:   179 |             console.log(formValues)
6:38:52 PM: error: Unexpected console statement (no-console) at src/views/Timer.vue:177:11:
6:38:52 PM:   175 |           console.log(newFocusTime)
6:38:53 PM: Shutting down logging, 53 messages pending
```

Commit:
some changes

- i was removed  the button pause from starts, it can be only settings screen
- fix lint errors

Signed-off-by: Lucas Matheus <mac@MacBook-Pro.local>

Commit:
simple changes
- altered datetime to only time
- the count of focus and break are removed

Signed-off-by: Lucas Matheus <mac@MacBook-Pro.local>

Commit:
Merge branch 'master' of github.com:RogaLabs/roga-timer
* 'master' of github.com:RogaLabs/roga-timer:
  Fix lint
  Add new specs for timer store module
  Add new items to the store
  Add vuetify
